.github/workflows/upload-artifact.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Do we want it for push or on tag create, i.e., 
```
on:  
  push:
    tags:
      - '*'
```

Reply:
Probably want it on tag create. This was just me testing how to create a VSIX in github workflows, and thought it might be helpful to have an intermediate artifact generated so i didnt have to create and share by hand.

Happy to put a pause on this while I write a script for automated release.

Should this logic live in here?
https://github.com/openfga/vscode-ext/blob/main/.github/workflows/publish.yaml

Reply:
Yes.  But lets take this for now while working on the automated release